frontend/src/pages/MapView/index.jsx

```diff
@@ -1,15 +1,34 @@
 import React, { Component } from "react";
-import ReactMapGL, { Marker, Popup } from "react-map-gl";
+import StaticMap, { Popup } from "react-map-gl";
 
 import { getSearchResults } from "../../utils/api";
-import Pin from "../../components/Pin";
 import ResourceCard from "../../components/ResourceCard";
 import Search from "../../components/SearchBar";
 import "./styles.scss";
+import DeckGL from "@deck.gl/react";
+import { IconLayer } from "@deck.gl/layers";
+import MarkerImg from "../../assets/images/marker.png";
 
-const pinSize = 35;
-
+const pinSize = 45;
 const searchSuggestions = [];
+const mapping = {
```

Comment:
not too much context on this so if there's any way to document what specifically this is doing, that would be nice

Reply:
Good idea. I'll add a comment here, but the idea is that DeckGL gives us this IconLayer, and it allows us to create a mapping between points of interest and different icons. For example, we might have different icons on our map in the future depending on the type of resource if that's something we can support. See [here](https://deck.gl/#/examples/core-layers/icon-layer) for example. Something to think about for future development work :)